SpringProject/src/test/java/com/GradProject2019/SpringProjectApplicationTests.java

```diff
@@ -1,4 +1,4 @@
-package com.GradProject2019.SpringProject;
+package com.GradProject2019;
```

Comment:
I assume you want both the SpringBootApplicationTests and SpringBootApplication files to be in the same package, so you'd probably want to rename this to `package com.gradproject2019`, too :)